src/Core/Settings.h

```diff
@@ -948,6 +947,7 @@ class IColumn;
 
 #define OBSOLETE_SETTINGS(M, ALIAS) \
     /** Obsolete settings that do nothing but left for compatibility reasons. Remove each one after half a year of obsolescence. */ \
+    MAKE_OBSOLETE(M, Bool, update_insert_deduplication_token_in_dependent_materialized_views, 1) \
```

Comment:
Previous default was `false`, why set to `1` now? Though this does not matter indeed

Reply:
I marked it as enabled because it is kinda enabled now always.